charts/opentelemetry-demo-datadog/otelcol-config.yaml

```diff
@@ -0,0 +1,198 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "otelcol"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              regex: ".*grpc_io.*"
+              action: drop
+  k8s_cluster:
+    collection_interval: 10s
+  hostmetrics:
```

Comment:
Make sure these are the same like [in the example](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d5bb86ce9ec34a62a6959ed22cc5bd61a6611215/exporter/datadogexporter/examples/collector.yaml#L16-L36)